src/widget/city.c

```diff
@@ -368,7 +368,16 @@ int widget_city_has_input(void)
 
 void widget_city_handle_mouse(const mouse *m)
 {
-    scroll_map(m);
+    if (m->right.is_down && !m->right.went_down) {
```

Comment:
I'm thinking this should probably go below the `if (m->is_touch)` line, otherwise mouse and touch will conflict.

Reply:
Good point.